README.md

```diff
@@ -25,3 +28,9 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+The runtime of the algorithm that I made was 1324ms which seems slow however I 
+ran it over an array of size 10,000,000. There are three merges into thirds giving
+us 3T(n/3), then the cost of merging them together is an additional constant 1. 
+Meaning the recurrence relation is T(n) = 3T(n/3) ignoring the constant factor. 
+Solving this equation gives us a runtime complexity of $\Theta$ = n.
```

Comment:
Please use the notation we've used in class -- $\Theta$ = n is syntactically meaningless.